plugins/services/src/js/pages/task-details/TaskDetailsTab.js

```diff
@@ -17,7 +17,7 @@ import Units from "#SRC/js/utils/Units";
 
 import MarathonTaskDetailsList from "../../components/MarathonTaskDetailsList";
 import TaskDirectoryStore from "../../stores/TaskDirectoryStore";
-import TaskEndpointsList from "../../components/TaskEndpointsList";
+// import TaskEndpointsList from "../../components/TaskEndpointsList";
```

Comment:
why not just remove the code @juliangieseke? I'm afraid this will be here for a long time.

Reply:
(hopefully) not, this is subtask #1 of its parent ticket, subtask #2 is also in this sprint, but i have to sync with dima what exactly we want to do.